### PR TITLE
fix(all): add required attributes in connect.yaml

### PR DESCRIPTION
**Pull request by leungkinghin-ct**

### CODE REVIEW

**Review by praveenkumarct — APPROVED** [review]

**Comment by industrian** [comment]
Hi. Please lowercase "commercetools" in all the `description` values. Thank you.

File: connect.yaml [hunk 1]
```diff
@@ -6,22 +6,31 @@ deployAs:
...
         - key: CTP_CLIENT_ID
           description: Commercetools client ID
+          required: true
         - key: CTP_SCOPE
           description: Commercetools client scope
+          required: true
         - key: CTP_AUTH_URL
           description: Commercetools Auth URL
+          required: true
         - key: CTP_API_URL
```
Comment (prateek-ct) [inline connect.yaml:19]: we can add europe gcp region as default value.

File: connect.yaml [hunk 2]
```diff
@@ -6,22 +6,31 @@ deployAs:
...
           description: Commercetools API URL
+          required: true
         - key: CTP_SESSION_URL
```
Comment (prateek-ct) [inline connect.yaml:22]: we can add europe gcp region session api url as default value.

File: connect.yaml [hunk 3]
```diff
@@ -6,22 +6,31 @@ deployAs:
...
           description: Session API URL
+          required: true
         - key: CTP_JWKS_URL
           description: JWKs url
```
Comment (prateek-ct) [inline connect.yaml:26]: we can add europe gcp region as default value.

File: connect.yaml [hunk 4]
```diff
@@ -6,22 +6,31 @@ deployAs:
...
+          required: true
         - key: CTP_JWT_ISSUER
           description: JWT Issuer for jwt validation
```
Comment (prateek-ct) [inline connect.yaml:29]: we can add europe gcp region as default value.

File: connect.yaml [hunk 5]
```diff
@@ -6,22 +6,31 @@ deployAs:
...
+          required: true
       securedConfiguration:
         - key: CTP_CLIENT_SECRET
           description: Commercetools client secret
+          required: true
   - name: enabler
```
Comment (prateek-ct) [inline connect.yaml:35]: Please move enabler entry before processor to clear cache during deployment as discussed this morning

**Review by prateek-ct — CHANGES_REQUESTED** [review]

File: connect.yaml (hunk 5 above)
Comment (leungkinghin-ct) [reply connect.yaml]: https://github.com/commercetools/connect-payment-integration-template/pull/171/commits/9070c015e2af5f71a21f37640635bcb18e0aef8d

File: connect.yaml (hunk 4 above)
Comment (leungkinghin-ct) [reply connect.yaml:35]: https://github.com/commercetools/connect-payment-integration-template/pull/171/commits/9070c015e2af5f71a21f37640635bcb18e0aef8d

File: connect.yaml (hunk 3 above)
Comment (leungkinghin-ct) [reply connect.yaml:31]: https://github.com/commercetools/connect-payment-integration-template/pull/171/commits/9070c015e2af5f71a21f37640635bcb18e0aef8d

File: connect.yaml (hunk 2 above)
Comment (leungkinghin-ct) [reply connect.yaml:26]: https://github.com/commercetools/connect-payment-integration-template/pull/171/commits/9070c015e2af5f71a21f37640635bcb18e0aef8d

File: connect.yaml (hunk 1 above)
Comment (leungkinghin-ct) [reply connect.yaml:22]: https://github.com/commercetools/connect-payment-integration-template/pull/171/commits/9070c015e2af5f71a21f37640635bcb18e0aef8d

File: connect.yaml [hunk 6]
```diff
@@ -6,22 +6,31 @@ deployAs:
...
+          required: true
         - key: CTP_CLIENT_ID
           description: Commercetools client ID
+          required: true
         - key: CTP_SCOPE
           description: Commercetools client scope
+          required: true
         - key: CTP_AUTH_URL
           description: Commercetools Auth URL
+          required: true
```
[the remark this answers is not in the document]
Comment (leungkinghin-ct) [reply connect.yaml:20]: https://github.com/commercetools/connect-payment-integration-template/pull/171/commits/9070c015e2af5f71a21f37640635bcb18e0aef8d

**Comment by leungkinghin-ct** [comment]
> Update connect.yaml

https://github.com/commercetools/connect-payment-integration-template/pull/171/commits/61288f5f3de9faf47ad8178b8c065f23b7bc0c90

**Review by prateek-ct — APPROVED** [review]

**Comment by leungkinghin-ct** [comment]
> Hi. Please lowercase "commercetools" in all the `description` values. Thank you.

https://github.com/commercetools/connect-payment-integration-template/pull/171/commits/61288f5f3de9faf47ad8178b8c065f23b7bc0c90

**Pull request merged**